tethys_gizmos/gizmo_options/cesium_map_view.py

```diff
@@ -366,23 +378,27 @@ class CesiumMapView(TethysGizmoOptions):
 
             object = 'link_to.glb'
 
-            models={'Cesium_Airplane': {
-                'model': {
-                    'uri': object1,
-                    'show': True,
-                    'minimumPixelSize': 128,
-                    'maximumScale': 20000,
-                    'shadows': 'enabled',
-                },
-                'name': object,
-                'orientation': {
-                    'Cesium.Transforms.headingPitchRollQuaternion': [
-                        {'Cesium.Cartesian3.fromDegrees': [-123.0744619, 44.0503706, 5000]},
-                        {'Cesium.HeadingPitchRoll': [{'Cesium.Math.toRadians' : 135}, 0, 0]}
-                    ]
+            models=[
```

Comment:
So something like this, with an MVLayer version instead of a dictionary with options and data:

```
models=[
    {'Cesium_Airplane': {
                'model': {
                    'uri': object1,
                    'show': True,
                    'minimumPixelSize': 128,
                    'maximumScale': 20000,
                    'shadows': 'enabled',
                },
                'name': object,
                'orientation': {
                    'Cesium.Transforms.headingPitchRollQuaternion': [
                        {'Cesium.Cartesian3.fromDegrees': [-123.0744619, 44.0503706, 5000]},
                        {'Cesium.HeadingPitchRoll': [{'Cesium.Math.toRadians' : 135}, 0, 0]}
                 },
                'position': {'Cesium.Cartesian3.fromDegrees': [-123.0744619, 44.0503706, 5000]},
    },
    MVLayer(
        options={'model': {
                        'uri': object1,
                        'show': True,
                        'minimumPixelSize': 128,
                        'maximumScale': 20000,
                        'shadows': 'enabled'},
                        'name': 'Cesium_Airplane',
                        'orientation': {
                            'Cesium.Transforms.headingPitchRollQuaternion':
                                [{'Cesium.Cartesian3.fromDegrees': [-123.0744619, 44.0503706, 5000]},
                                 {'Cesium.HeadingPitchRoll': [{'Cesium.Math.toRadians': 135}, 0, 0]}]},
                        'position': {'Cesium.Cartesian3.fromDegrees': [-123.0744619, 44.0503706, 5000]}
                    },
        data={
                    'layer_id': "cesium_airplane_id",
                    'layer_name': "Cesium_Airplane",
                    'popup_title': "Cesium Airplane"}
                },
    )
]

```